packages/zero-cache/src/services/view-syncer/view-syncer.ts

```diff
@@ -617,8 +620,8 @@ export class ViewSyncerService implements ViewSyncer, ActivityBasedService {
         }
 
         // Apply requested patches.
+        lc.debug?.(`applying ${desiredQueriesPatch?.length} query patches`);
```

Comment:
his `har` file has `desiredQueriesPatches` so this shouldn't be the issue but moving the log out of the `if` just in case.